<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manageUsers</class>
 <widget class="QDialog" name="manageUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Manage user accounts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Press delete/add button to delete/add an account</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Search for users:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="searchUser"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="listAccounts">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteUserButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addUserButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>manageUsers</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
